mcs/class/System.Net.Http/CFNetworkHandler.cs

```diff
@@ -100,7 +100,7 @@ void EnsureModifiability ()
 
 		public CookieContainer CookieContainer {
 			get {
-				return cookies;
+				return cookieContainer ?? (cookieContainer = new CookieContainer ());
```

Comment:
yes, typo :( and we have something broken in System.Net.Http dependencies (XI build is special) so it did not break locally